### ISSUE TITLE
Package(s) installed/removed/updated as description

### ISSUE THREAD

**Issue by Magissia** [stefxh/apt-btrfs-snapper#4]
Congratulations, your script is available officially on canonical's repos !

Would be nice to have affected package in the description instead of just "apt".

**Comment by bmullan** [comment]
I totally agree wtih this... why not put the apt command & packages installed into the Description field
Having possibly many entries of:
- pre
- timeline
- post

with no idea "what" the specific snapshot was for doesn't help anyone...
